### DISCUSSION TITLE
Another SMTP

### DISCUSSION THREAD

**Discussion by AndersonOuverney** [postalserver/postal#1616] [Help with using Postal]
Hello good afternoon.
Currently we are using the postal  very well, however, at times it happens that we receive many messages in SMTP, this ends up creating a queue at times.

I believe that if we had another SMTP service to receive and queue messages we could drastically improve our speed.

So I ask, what would be the best way for us to perform an additional installation of just the SMTP service and everything it needs to work? We want to keep the existing database.

Thank you very much

**Comment by willpower232** [comment]
Are you using Postal v2? Do you need to scale up the worker count?

**Reply by AndersonOuverney** [reply]
Hello good Morning.
I did a test install of postal v2.

My need would be to increase the amount of SMTP services that process the messages that arrive at thepostal, not those that are sent through the workers.

Our goal is to increase the capacity to receive the postal and only then deliver the messages

**Reply by willpower232** [reply]
When you say there is a queue, is that in your application trying to send the emails or in Postal trying to be sent?

**Reply by AndersonOuverney** [reply]
Hello Will good morning.
Queues are in applications trying to send to the postal,
We have a timeout error, I believe this is because we only have 1 STMP service

At times we also have a queue at the postal trying to send to the final recipients, but here would be another question.

**Reply by willpower232** [reply]
Interesting, how many messages are you trying to send? It might be worth having a go with the API rather than SMTP so the requests are a little easier to make.
